01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs').promises;
+
+async function writer() {
+    try{
+        const data = `Line 1\nLine 2\nLine 3`;
+        await fs.writeFile ('../answers/temporary/tempt.txt', data);
+        console.log('File tempt.txt has been written succesfully.')
+
+    } catch(error){
+        console.log('Error with file:', error)
+    }
+}
```

Comment:
Ideally you would return the await writeFile and have two separate writeFile statements. You don't have to correct but good thing to note.

You didn't implement the other two functions.

 await writeFile("temp.txt","Line 1");
  await writeFile("temp.txt","Line 2\n", {flag: "a"});
   var test = await writeFile("temp.txt","Line 3", {flag: "a"});
   return test // where test would be a promise